README.md

```diff
@@ -332,6 +333,7 @@ Contains the request's input data
 |`_id`|`string`| Document unique identifier |
 |`action`|`string`| Kuzzle action to perform |
 |`body`|`object`| Contains request specific data (document content, search queries, ...) |
+| `headers` | `object` | request headers (ie. HTTP headers for HTTP an Websocket protocols) |
```

Comment:
Proposal: `request input headers (e.g. HTTP headers for HTTP or Websocket protocols)`